themes/classic/templates/catalog/_partials/miniatures/product.tpl

```diff
@@ -70,7 +70,10 @@
               {hook h='displayProductPriceBlock' product=$product type="before_price"}
 
               <span class="sr-only">{l s='Price' d='Shop.Theme.Catalog'}</span>
-              <span itemprop="price" class="price">{$product.price}</span>
+              <span itemprop="offers" itemscope itemtype="http://schema.org/Offer">
```

Comment:
Can you use https version instead?

Reply:
After checking in the schema.org faq, they mention it works either way and will migrate to https eventually. So I did change to https.
Also changed another http to https in the same file.